project/build.properties

```diff
@@ -1,2 +1,2 @@
 # Remember to change the sbt version in all sample projects when updating it here
-sbt.version=0.13.18
+sbt.version=1.4.5
```

Comment:
is there a reason not to go all the way to 1.4.7?

Reply:
no, I've updated